### DISCUSSION TITLE
Can I get all kind of ops latency after running a model?

### DISCUSSION THREAD

**Discussion by xpmemeda** [microsoft/onnxruntime#19326] [Performance Q&A] [answered]
What I want is a table like this:

![image](https://github.com/microsoft/onnxruntime/assets/39027942/4cae9497-5ba2-4d21-b147-9727d9e21b77)

**Comment by skottmckay** [comment, marked as answer]
This will output per-node info that you can process if you need additional statistics. The overall execution time will be meaningless due to the cost of producing this output so ignore that.

https://onnxruntime.ai/docs/performance/tune-performance/profiling-tools.html#in-code-performance-profiling
